install/snipeit-install.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+#Copyright (c) 2021-2024 community-scripts ORG
+# Author: Michel Roegl-Brunner (michelroegl-brunner)
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+source /dev/stdin <<<"$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y curl
+$STD apt-get install -y git
+$STD apt-get install -y sudo
+$STD apt-get install -y mc
+$STD apt-get install -y nginx
+$STD apt-get install -y php8.2-{bcmath,common,ctype,curl,fileinfo,fpm,gd,iconv,intl,mbstring,mysql,soap,xml,xsl,zip,cli}
+$STD apt-get install -y mariadb-server
```

Comment:
```bash
$STD apt-get install -y \
  curl \
  git \
  sudo \
  mc \
  nginx \
  php8.2-{bcmath,common,ctype,curl,fileinfo,fpm,gd,iconv,intl,mbstring,mysql,soap,xml,xsl,zip,cli} \
  mariadb-server 
```

is the smarter way, is realy every of this php8.2 packages needed?

Reply:
Yep, this is the better way. And yes, all this extentions are needed per requirements. (https://snipe-it.readme.io/docs/requirements)